generators/Exercises/PascalsTriangle.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using Generators.Input;
+using Generators.Output;
+using Newtonsoft.Json.Linq;
+
+namespace Generators.Exercises
+{
+    public class PascalsTriangle : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            foreach (var dataCases in canonicalData.Cases)
+            {
+                dataCases.UseVariableForExpected = true;
+                dataCases.Property = "calculate";
+                if (!(dataCases.Expected is JArray))
+                    dataCases.ExceptionThrown = typeof(ArgumentOutOfRangeException);
+                if (dataCases.Properties["count"] == null)
+                    dataCases.Properties["count"] = -1;
+            }
```

Comment:
Should we be converting this to a -1? It should be entirely possible to pass in null (int?), or pass in nothing with a default param.

Reply:
@jpreese Well, I think the canonical data is at fault here. In what way would passing null make sense for calculating pascal's triangle? I've created [an issue](https://github.com/exercism/problem-specifications/issues/925) to discuss removing this strange case. In the mean time, I'm in favor of manually removing this error case from the canonical data to make sure we don't output it.

Reply:
@jpreese  @bmeverett I've created a [PR](https://github.com/exercism/problem-specifications/pull/926) to fix the canonical data.

Reply:
Yeah, I'd agree. I know there was a PR over in the problem-specs repo about wanting to test edge cases (e.g. null). I'm not sure if this is a result of that or where that got left off.